NOTICE

```diff
@@ -25,6 +25,35 @@ OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 ================================================================================
 The MIT License applies to:
 
+* The tmTheme in `assets/files/halcyon.tmTheme`
```

Comment:
You can merge this entry with the other MIT License, see egg SIL License for how it's done in other places

Reply:
It has the copyright `Copyright (c) 2018 Brittany Chiang`. Can I remove the copyright and merge it the other MIT License?

Reply:
For the SIL License, there are also different copyrights. We keep them in the list next to the affected files.